playbooks/common/openshift-cluster/upgrades/pre/verify_control_plane_running.yml

```diff
@@ -9,23 +9,16 @@
       local_facts:
         ha: "{{ groups.oo_masters_to_config | length > 1 }}"
 
-  - name: Ensure Master is running
-    service:
-      name: "{{ openshift.common.service_type }}-master"
-      state: started
-      enabled: yes
-    when: openshift.master.ha is defined and not openshift.master.ha | bool and openshift.common.is_containerized | bool
-
   - name: Ensure HA Master is running
     service:
       name: "{{ openshift.common.service_type }}-master-api"
       state: started
       enabled: yes
-    when: openshift.master.ha is defined and openshift.master.ha | bool and openshift.common.is_containerized | bool
+    when: openshift.common.is_containerized | bool
```

Comment:
I think that *is* what we want - upgrade to be blocked.

Technically, the support path for this old mode (which is unusual) should be removed.  We don't support upgrade from non-HA to HA.